libs/designsystem/src/lib/components/modal/services/modal.helper.ts

```diff
@@ -48,7 +48,7 @@ export class ModalHelper {
         'kirby-modal',
         config.flavor === 'drawer' ? 'kirby-drawer' : null,
         config.flavor === 'compact' ? 'kirby-modal-compact' : null,
-        modalSize,
+        'kirby-modal--' + modalSize,
```

Comment:
Please align with existing naming:
```suggestion
        'kirby-modal-' + modalSize,
```